src/config/test.ts

```diff
@@ -1,5 +1,5 @@
 export default {
-  db: 'mongodb://Admin:passwordforadmin531@ds163494.mlab.com:63494/car-finder',
+  db: 'mongodb://test:testpass@ds121965.mlab.com:21965/car-finder-test',
```

Comment:
In a real-world scenario, such variables must never go to github.
As you're sharing a super sensitive data with a 3-rd party.